guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Creating_an_Image_for_Amazon_EC2_{context}"]
+= Creating an Image for Amazon EC2
+
+You can create images for Amazon EC2 from within {Project}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Infrastructure > Compute Resources*.
+. Select your Amazon EC2 provider.
+. Click *Create Image*.
+
+* In the *Name* field, enter a meaningful and unique name for your EC2 image.
+* In the *Operating System* drop down menu, associate the image to an operating system within {Project}.
+* In the *Architecture* drop down menu, associate the image to an architecture within {Project}.
```

Comment:
```suggestion
* From the *Operating System* list, select an operating system to associate with the image.
* From the *Architecture* list, select an architecture to associate with the image.
```
If I am not mistaken, *list* is the preferable name for drop down menu. Also, does the list include only operating systems/architectures within {Project}? If yes, that would mean that "within {Project}" could be dropped.

Reply:
Applied.